volatility/framework/plugins/linux/check_idt.py

```diff
@@ -0,0 +1,94 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+from typing import List
+
+from volatility.framework import interfaces, renderers, constants, contexts, exceptions, symbols
+from volatility.framework.configuration import requirements
+from volatility.framework.renderers import format_hints
+from volatility.framework.symbols import linux
+from volatility.plugins.linux import lsmod
+
+vollog = logging.getLogger(__name__)
+
+
+class Check_idt(interfaces.plugins.PluginInterface):
+    """ Checks if the IDT has been altered """
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        return [
+            requirements.TranslationLayerRequirement(name='primary',
```

Comment:
Our yapf configuration puts spaces between `=` in parameters.  Happy to consider changing the yapf style file, but it'd be good to get things formatted using it, given it should do it all for us.  5:)

Reply:
I feel spaces make the code look cleaner so I'll fix that up